java/maven/src/org/netbeans/modules/maven/execute/DefaultReplaceTokenProvider.java

```diff
@@ -215,8 +214,13 @@ private static FileObject[] extractFileObjectsfromLookup(Lookup lookup) {
                 HashSet<String> test = new HashSet<String>();
                 addSelectedFiles(false, fos, test);
                 addSelectedFiles(true, fos, test);
-                String files2test = test.toString().replace(" ", "");
-                packClassname.append(files2test.substring(1, files2test.length() - 1));
+                
+                packClassname.append(test
+                        .stream()
+                        .map(String::trim)
+                        .collect(Collectors.joining(","))
```

Comment:
Please check whether syntax with comma-separated package names (not paths) works in ancient Surefire version - according to docs, there were some significant changes in Surefire 2.19. We recommend Surefire 2.22 for Junit5, and surefire 2.8 (which may fail with this syntax) for single test run. See `org.netbeans.modules.maven.ActionProviderImp::checkSurefire` for details.

Reply:
I made a test with junit:4.12 and maven-surefire-plugin:2.8. In Netbeans I selected 2 packages and execute them. It generated `-Dtest=some.it.pkg.other.**,some.it.pkg.**` and it run the tests in those packages as expected:

`
Running some.it.pkg.ItTest
...
Running some.it.pkg.other.It3Test
...`

Is a such manual test enough?